db/migrate/20210901192909_update_vaccine_product_names_to_match_cdc.rb

```diff
@@ -0,0 +1,47 @@
+    Vaccine.where(product_name: 'Sinopharm (BIBP) COVID-19 Vaccine (Non-US)').update_all(product_name: 'Sinopharm (BIBP) COVID-19 Vaccine')
+
+    # Update all saved user_export_presets
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_export_presets SET config = REPLACE(config, 'Moderna COVID-19 Vaccine (Non-US tradename: Spikevax)', 'Moderna COVID-19 Vaccine (non-US Spikevax)') WHERE config REGEXP 'Moderna'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_export_presets SET config = REPLACE(config, 'Pfizer-BioNTech COVID-19 Vaccine (Tradename: COMIRNATY)', 'Pfizer-BioNTech COVID-19 Vaccine (COMIRNATY)') WHERE config REGEXP 'Pfizer'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_export_presets SET config = REPLACE(config, 'AstraZeneca COVID-19 Vaccine (Non-US tradenames: VAXZEVRIA, COVISHIELD)', 'AstraZeneca COVID-19 Vaccine (Non-US tradenames include VAXZEVRIA, COVISHIELD)') WHERE config REGEXP 'Pfizer'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_export_presets SET config = REPLACE(config, 'Coronavac (Sinovac) COVID-19 Vaccine (Non-US)', 'Coronavac (Sinovac) COVID-19 Vaccine') WHERE config REGEXP 'Pfizer'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_export_presets SET config = REPLACE(config, 'Sinopharm (BIBP) COVID-19 Vaccine (Non-US)', 'Sinopharm (BIBP) COVID-19 Vaccine') WHERE config REGEXP 'Pfizer'")).execute
+
+    # Update all saved user_filters
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_filters SET contents = REPLACE(contents, 'Moderna COVID-19 Vaccine (Non-US tradename: Spikevax)', 'Moderna COVID-19 Vaccine (non-US Spikevax)') WHERE contents REGEXP 'Moderna'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_filters SET contents = REPLACE(contents, 'Pfizer-BioNTech COVID-19 Vaccine (Tradename: COMIRNATY)', 'Pfizer-BioNTech COVID-19 Vaccine (COMIRNATY)') WHERE contents REGEXP 'Pfizer'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_filters SET contents = REPLACE(contents, 'AstraZeneca COVID-19 Vaccine (Non-US tradenames: VAXZEVRIA, COVISHIELD)', 'AstraZeneca COVID-19 Vaccine (Non-US tradenames include VAXZEVRIA, COVISHIELD)') WHERE contents REGEXP 'Pfizer'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_filters SET contents = REPLACE(contents, 'Coronavac (Sinovac) COVID-19 Vaccine (Non-US)', 'Coronavac (Sinovac) COVID-19 Vaccine') WHERE contents REGEXP 'Pfizer'")).execute
+    ActiveRecord::Base.connection.raw_connection.prepare(Arel.sql("UPDATE user_filters SET contents = REPLACE(contents, 'Sinopharm (BIBP) COVID-19 Vaccine (Non-US)', 'Sinopharm (BIBP) COVID-19 Vaccine') WHERE contents REGEXP 'Pfizer'")).execute
```

Comment:
the regexp filter on these is for pfizer, not the respective vaccine name

Reply:
good catch

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1064/commits/11b2af435b70857d818dac5163892c81e2ea6f2f